source/solvers/gls_sharp_navier_stokes.cc

```diff
@@ -287,7 +294,7 @@ GLSSharpNavierStokesSolver<dim>::force_on_ib()
           unsigned int p;
           bool         cell_is_cut;
           std::tie(cell_is_cut, p) = cut_cells_map[cell];
-          // If the cell is cute
```

Comment:
I always liked cute cells

Reply:
Me too to be honest. But then it's discrimination for not cute cells so